lib/src/configuration.dart

```diff
@@ -91,6 +97,11 @@ class ExplicitConfiguration extends Configuration {
   ExplicitConfiguration(Map<String, ConfiguredValue> values, this.nodeWithSpan)
       : super.implicit(values);
 
+  /// ID for a [configuration] created with an explicit `@use ... with` uniquely
+  /// identified by its [nodeWithSpan] properties.
+  @override
+  int get opaqueId => nodeWithSpan.span.toString().hashCode;
```

Comment:
Are there situations where we have multiple distinct `Configuration` objects from the same node that we want to consider equivalent?

Reply:
When testing this every time a module was multiple times with "the same config" it was actually behaving as if the `Configuration` object was a completely different instance/reference (as pointed above this might have been my misunderstanding on how `identityHashcode` works, but.. assuming my understanding _was_ correct then this is the reason I was hoping to consider this value equivalent

But I'll double check this :)

Reply:
double checked and that's indeed the behavior, the instances are different even though it's the same config